libraries/Microsoft.Bot.Builder.AI.QnA/StrictFiltersCompoundOperationType.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.AI.QnA
+{
+    /// <summary>
+    /// Operation types for Search Filter.
```

Comment:
this is not search filter. please fix the comments

Reply:
implemented the change

---
In reply to: [470570672](https://github.com/microsoft/botbuilder-dotnet/pull/4446#discussion_r470570672) [](ancestors = 470570672)